projects/LandMine.md

```diff
@@ -0,0 +1,94 @@
+Application
+
+## Project Description
+<!--Describe your project in a few sentences. -->
+Landmine is a platform where each property {like apartments,bunglow} is represented as an nft and then these nft are put up for sale.
+-Buyers can buy the properties listed on the site.
+-After Buyer agress to buy the property they will upload the agreement details which will be saved in the ipfs platform and seller will complete the transaction after the completion of payment.
+-Buyer can verify these transaction details by using their property id by entering their id and they will get the ipfs link of their agreement.
+
+### Use of IPFS
+- We have used ipfs for storing the documents related to the agreement details .
+- here is the <a href="https://gateway.ipfs.io/ipfs/QmTiKQjNEAeNpxdkMNvWpKj4jbz8cRaEp4EjQAyAHYURAT">link</a>
+ for the demo agreement details made by using ipfs.
+
+
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+Need to add the registration page, redesign the ui and the transaction page and little tweaks in the smart contracts regarding the revenue.
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+Screenshots: ![alt text](https://siasky.net/vAHX05cSzk8_3diwKLFumx4lGXFN3M0cgVAioQDQxdjrIg)
+
+
+![alt text](https://siasky.net/LADILo82OTgI6clQsVhn5w58BVwqpB_QVFJ7Vg8zpEz1FQ)
+
+
+![alt text](https://siasky.net/jADE_NAY6CdapliFLhKSFY6zQwAFXblvLxS3Q4WqdvIHPw)
+
+
+![alt text](https://siasky.net/LAC6uvrtyQzAPYqBXMiSAiq0Fx_8kUGgg1EWZxxOsd0U3Q)
+
+
+![alt text](https://siasky.net/DADm7GaFYmjPhP8eM7a2FmN61eOcFvpBY3TC_P7kLDz5ag)
+ ![alt text](https://siasky.net/fAb9RSi49mIgDIRlLb4qy_cHAsRPub1rYY2yXeSmr07Aqw)
+
+![alt text](https://siasky.net/fABWmFWSi_B-VcxYXNhjFQPDoJ5YPfEVm5tn3ioaAEV_Mw)
+![alt text](https://siasky.net/zACIz4ziCpjSOpy4OQcHeFBL2i1DAldxB_aGqMBoGe1lnQ)
+
+![alt text](https://siasky.net/vAOlzpNoPqCbA6QvK7MZmWBxqpe-LLA_qYjA9kUZjbXJVQ)
+
+              
+Video: https://youtu.be/QuhAO8GlKT8
+
+## Target Audience
+<!--Describe who will be your project's users-->
+General public interested in buying and selling property.
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+N/A
+
+## Github repo
+<!--Attach a link to your GitHub repo if it's OSS-->
+https://github.com/Mr-Maximus/landmine_project
+
+## Website
+<!--Link your website if available-->
+TBA
+
+## Docs
+<!--Including a link to your project docs!-->
+In progress
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+The LandMinw team was assembled as part of the Chainlink hackathon there are currently 2 members
+
+### Team Size
+2 members
+
+### Team members
+[Madan Raj.S](https://github.com/Mr-Maximus)
+[Mukesh Jaiswal](https://github.com/MukeshJaiswal01)
+
+
+## How the community can engage
+GitHub Discussion:https://github.com/ipfs/community/discussions/517
```

Comment:
This is discussion for different project, can you add one for yours?

Reply:
Sorry my bad . I have updated it now . here is the <a href="https://github.com/ipfs/community/discussions/529">link</a>